spell/spell-reviewer-goerli-checklist.md

```diff
@@ -243,7 +243,10 @@ Spell Actions (Per Exec Sheet):
 * [ ] Tests
   * [ ] Ensure each spell action has sufficient test coverage
     _List actions for which coverage was checked here_
-  * [ ] Ensure every test function is declared as public if enabled or private if disabled
+  * [ ] Ensure every test function is declared as `public`
+    * [ ] IF the test is enabled, it MUST NOT have the `skipped` modifier
+    * [ ] OTHERWISE, if the test is disabled, it MUST have the `skipped` modifier
```

Comment:
I would reformulate those checks OR add another check (below `Check all tests are passing locally using make test`) to ensure that every test function listed inside `Ensure each spell action has sufficient test coverage` is present in the test output with `PASS` prefix. Because with new prefixes and similar color coding (green/yellow) it might be confusing, especially when conditional skip is applied via `skippedWhenDeployed`/ `skippedWhenNotDeployed`

Reply:
I pushed some changes, please check again now.